### DISCUSSION TITLE
What to backup for later restoration?

### DISCUSSION THREAD

**Discussion by danielcharrua** [ACINQ/phoenixd#109] [Q&A] [answered]
Hi @pm47 and phoenix team!

I'm creating some guides and wanted to know what to backup to restore in case of emergency.

I think that doing some backup to the `.phoenix` folder will be enough, but I prefer to ask before.

Anything else? Is there a secure way to backup this folder containing sensitive information? Maybe encrypting and then backup?

Thank you

**Comment by pm47** [comment, marked as answer]
Hello,

The **minimal backup** is the seed (`seed.dat`). With only the seed you can recover your channel and your balance, but you will lose the payment history.

The **full backup** is the seed (`seed.dat`) + the database (`phoenix.xxx.db`). It will allow you to recover everything.

Backuping the whole `.phoenix` folder is a superset of the full backup, you can choose to do that for simplicity. However, a good security practice would be to backup the seed (which never changes and is very sensitive) separately from the database (which is less sensitive but needs to be periodically refreshed).
